### DISCUSSION TITLE
How to emit examples in json schema?

### DISCUSSION THREAD

**Discussion by risabhds** [microsoft/typespec#3944] [Json Schema]
I have the following typespec in my [playground](https://typespec.io/playground?c=aW1wb3J0ICJAdHlwZXNwZWMvanNvbi1zY2hlbWEiOwoKdXNpbmcgVHlwZVNwZWMuSnNvblPFHTsKCkDELMYOCm5hbWVzcGFjZSDGEXM7CgovKiogRGVzY3JpcHRpb24gZ29lcyBoZXJlICovCkBzdW1tYXJ5KCJUaXRsZcodIikKbW9kZWwgUGVyc29uIHsKICDESlRoZSBwxRMncyDEby7ERyAgQGV4YW1wbGUoIkpvaG4gRG%2FEQiDFIjogc3RyaW5nOwrHREFnZSBpbiB5ZWFycyB3aGljaCBtdXN0IGJlIGVxdWFsIHRvIG9yIGdyZWF0ZXIgdGhhbiB6ZXJv0GsxOMRjQG1pblZhbHVlKDDED2FnZTogaW50ZWdlcjsKfQo%3D&e=%40typespec%2Fjson-schema&options=%7B%22options%22%3A%7B%22%40typespec%2Fjson-schema%22%3A%7B%22file-type%22%3A%22json%22%2C%22int64-strategy%22%3A%22number%22%7D%7D%7D):

```typescript
import "@typespec/json-schema";

using TypeSpec.JsonSchema;

@jsonSchema
namespace Schemas;

/** Description goes here */
@summary("Title goes here")
model Person {
  /** The person's name. */
  @example("John")
  name: string;

  /** Age in years which must be equal to or greater than zero. */
  @example(18)
  @minValue(0)
  age: integer;
}
```

But I don't get `examples` in the emitted json-schema.

What I desire is a json-schema with `examples` on the property:
```json
{
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "$id": "Person.json",
    "description": "Description goes here",
    "title": "Title goes here",
    "type": "object",
    "properties": {
        "name": {
            "type": "string",
            "description": "The person's name.",
            "examples": ["John"]
        },
        "age": {
            "type": "integer",
            "minimum": 0,
            "description": "Age in years which must be equal to or greater than zero.",
            "examples": [18]
        }
    },
    "required": [
        "name",
        "age"
    ]
}
```

How do I get `examples` in the emitted json-schema?

**Comment by timotheeguerin** [comment]
json schema emitter doesn't yet make use of the example decorator. Could convert that to an issue

**Reply by risabhds** [reply]
Thanks for the prompt response!
Created the issue #3950.

**Reply by risabhds** [reply]
Just noticed that you already created the issue https://github.com/microsoft/typespec/issues/3946. Thanks!
Closed my issue as duplicate.

**Comment by timotheeguerin** [comment]
@risabhds following back on this, is there actually any specification on `examples` in json schema? I can't seem to find it? If not I am not sure we should be adding that at least for now.

**Reply by timotheeguerin** [reply]
Ignore, found it https://json-schema.org/draft/2020-12/json-schema-validation#name-examples
